### PR TITLE
174099259: Add SameSite attribute to auth cookies

### PR DESCRIPTION
**Pull request by mkleymenov**
**Context**
Browsers have started rolling out changes to SameSite cookie policy that default `SameSite=Lax` to cookies that do not specify that attribute explicitly. The default policy prevents cookies from being sent in cross domain requests. This in particular breaks Zendesk integration with Meetup Admin, due to Google SSO cookies not being passed from Zendesk to the admin widget.
image

**What does this PR do?**
First a note that a project this repository is based upon have been deprecated and is no longer maintained: https://github.com/cloudflare/nginx-google-oauth.

This PR adds `SameSite=None` attribute to cookies set by gauth-proxy nginx plugin (namely, `OauthEmail`, `OauthAccessToken` and `OauthExpires`) by rewriting cookies path with [proxy-cookie-path](http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_cookie_path) Nginx directive. In addition, `Secure` cookie flag is forced by default.

In addition, this PR does a bit of maintenance:
* Adds a service definition for Pantry
* Adds `latest` tag for the latest Docker images

### CODE REVIEW

File: default.conf [hunk 1]
```diff
@@ -39,7 +39,6 @@ server {
   set_by_lua $ngo_token_secret    'return os.getenv("NGO_TOKEN_SECRET")';
   set_by_lua $ngo_secure_cookies  'return os.getenv("NGO_SECURE_COOKIES")';
   set_by_lua $ngo_extra_validity  'return os.getenv("NGO_EXTRA_VALIDITY")';
-  set_by_lua $ngo_extra_validity  'return os.getenv("NGO_EXTRA_VALIDITY")';
```
Comment (mkleymenov) [inline default.conf:42]: Duplicate line

File: default.conf [hunk 2]
```diff
@@ -58,5 +57,6 @@ server {
     proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header X-Forwarded-Proto $scheme;
     proxy_set_header Google-User $ngo_user;
+    proxy_cookie_path / "/; SameSite=None";
```
Comment (mkleymenov) [inline default.conf:60]: http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_cookie_path

File: Makefile [hunk 3]
```diff
@@ -29,7 +29,9 @@ component-test:
 
 #Pushes the container to the docker registry/repository.
 publish: package
-	@docker push $(PUBLISH_TAG)
+	docker tag meetup/gauth-proxy:latest $(PUBLISH_TAG)
```
Comment (chenrui333) [inline Makefile:32]: I dont recommend to use `latest` in the client side. But it should be good just pushing this alias.

**Review by chenrui333 — APPROVED** [review]
lgtm

**Pull request merged**